experiments/ClimaEarth/user_io/postprocessing.jl

```diff
@@ -72,7 +73,6 @@ function postprocess_sim(::Type{AMIPMode}, cs, postprocessing_vars)
 
     # Check this because we only want monthly data for making plots
     if t_end > 84600 * 31 * 3 && output_default_diagnostics
-        include("leaderboard/leaderboard.jl")
```

Comment:
Can we also move `include("user_io/diagnostics_plots.jl")` into global scope? Julia does not permit loading code inside functions like that.

Reply:
And below (`include("user_io/diagnostics_plots.jl")`)

Reply:
Ah sorry, I didn't see all the includes inside the functions. Fixed now.

Reply:
It looks like the includes were not added, they were only removed

Reply:
It seems `diagnostics_plots.jl` is already included at the top of this file.